changelog/upgrade-notes.md

```diff
@@ -11,6 +11,32 @@ This version of OpenLayers is only compatible with `ol-mapbox-style@12.4.0` or h
 Returning false from the listener function will now stop propagation, when the listener is added with `once`.
 Previously this only worked with the `on` method.
 
+#### The `filter` option for `WebGLPointsLayer` has changed
+
+The filter option for the `WebGLPointsLayer` must now be specified alongside other options instead of being part of the `style` object.
```

Comment:
```suggestion
The filter option for the `WebGLPointsLayer` must now be specified alongside other options instead of being part of the `style` object. Note that the `WebGLPointsLayer` is not part of the stable API and is subject to breaking changes between major releases.
```